tokio-fs/src/os/windows/mod.rs

```diff
@@ -0,0 +1,7 @@
+//! Windows-specific extensions for the primitives in the `tokio_fs` module.
+
+mod symlink_dir;
+mod symlink_file;
+
+pub use symlink_dir::{symlink_dir, SymlinkDirFuture};
```

Comment:
Missing `self::`, maybe? This failed on AppVeyor.

Reply:
It was two missing `self::`